### ISSUE TITLE
EEZ flow: In Plot: Line  labels not shown in run

### ISSUE THREAD

**Issue by fietser28** [eez-open/studio#197]
In Run or Debug mode/window. Trace 0 - 4 is shown instead of the line labels:
<img width="1066" alt="image" src="https://user-images.githubusercontent.com/8796121/184560653-4e9a16fe-803c-496d-9230-02d570b69b3b.png">

In Edit mode/window (these are the set names of the lines):
<img width="1418" alt="image" src="https://user-images.githubusercontent.com/8796121/184560737-c476496f-9dc7-4a8d-9c37-b894439535ed.png">

A dashboard project using 0.9.94 on Mac M1.

**Comment by mvladic** [comment]
Can't reproduce. Could you please attach a simplified dashboard project with this problem?

Here is my testing dashboard project:

https://1drv.ms/u/s!AjzKAPRhTu0CpLR7aGmp3A80cslLRQ?e=8pcuZC

Here is the screenshot in Run mode:

![image](https://user-images.githubusercontent.com/449788/184621967-0594c66b-b3a1-4e71-afaf-4cb3ef408694.png)

**Comment by fietser28** [comment]
I've modified your project file (attached in this message) to produce the error:
I think it has to do with the reset input:  The first time you press the start button  the labels are wrong. On subsequent presses the labels are correct.
I've also played with the X value but that doesn't seem to make any difference.

[test_line_chart_w_reset.eez-project.zip](https://github.com/eez-open/studio/files/9339230/test_line_chart_w_reset.eez-project.zip)

<img width="862" alt="image" src="https://user-images.githubusercontent.com/8796121/184698473-f6561a86-8cee-4d08-979b-8d1b13c83a07.png">
<img width="862" alt="image" src="https://user-images.githubusercontent.com/8796121/184698607-ea61192b-4877-4cc8-ad1b-c75b9847096c.png">

**Comment by mvladic** [comment]
Should be fixed in latest build (0.9.95).

**Comment by fietser28** [comment]
The previous example is working with today's build.

The attached example is not (anymore?):
- With a single trace the label is not shown
- Data.now() doesn't plot any data anymore. A constant or variable works as expected.

[PlotTest.eez-project.zip](https://github.com/eez-open/studio/files/9385206/PlotTest.eez-project.zip)

**Comment by mvladic** [comment]
I suppose you mean `Date.now()`, not `Data.now()`? 

I can confirm that first point doesn't work, but second point I can't repeat since I can see date's on X axis:

![image](https://user-images.githubusercontent.com/449788/185788557-9345c9ad-d9da-45cf-bfe5-0c25e6de5849.png)

**Comment by mvladic** [comment]
It appears that Plotly library chooses not to show legend if there is only a single trace line. Luckily, we can override this behavior if we set its `showlegend` property to `true`. I added a new property to LineChart widget called "Show legend", by default it is set to `true`. If set to `false` it will not only hide legend for single trace charts, but for all charts.

![image](https://user-images.githubusercontent.com/449788/185788928-dbecb45c-6541-44d1-bf2a-23dc38390241.png)
